clowder-framework/clowder
# Fixed incorrect selector on imagesLoaded in the Space view

## Description

### Problem
When `collectionsBySpace` is empty and ` datasetsBySpace` is not, the dataset list can occasionally render on top of the empty collections list.

### Approach
* ~~Wrapped `datasetsBySpaces` and `collectionsBySpaces` with a new row~~ this did not resolve the issue
* ~~Added a slight delay before `masonry` activates in the Space view to allow images to load first~~ this is very fragile/hacky
* Fixed an incorrect selector on `imagesLoaded` that may have been preventing masonry from firing at the correct time after images had loaded

### How to Test
Prerequisites: at least one dataset, at least one file uploaded to the dataset, image-preview extractor has run on at least one file in the dataset

Setup:
1. Create a space and add a dataset to it
2. Enable RabbitMQ plugin, run RabbitMQ, run `image-preview` extractor
3. Upload one or more image files to the dataset
    * This should generate a preview on the file, as well as on the dataset itself

Test Case:
1. Checkout and run this branch locally
2. Login to Clowder
3. Navigate to the space
    * You should see the Datasets (not empty) and Collections (empty) sections here
4. Refresh the page a few dozen times
    * You should see that the Datasets section does not overlap the Collections section

## Review Time Estimate


## Types of changes


## Checklist:


- [x] I have updated the documentation accordingly. (N/A)
- [x] I have added tests to cover my changes. (N/A)
- [x] All new and existing tests passed. (N/A)
